examples/typescript/bulkDisableSchedules.ts

```diff
@@ -7,28 +7,27 @@ const sdk = LookerNodeSDK.init40()
 
 const getSchedules = async (allUsers: boolean, userID?: number): Promise<IScheduledPlan[]> => {
     let schedules: IScheduledPlan[];
-    if (allUsers) {
-        schedules = await sdk.ok(sdk.all_scheduled_plans({all_users:true}))
+    if(userID) {
+        schedules = await sdk.ok(sdk.all_scheduled_plans({user_id: userID, all_users:false}))
     } else {
-        if (!userID) {
-            schedules = undefined
-            throw new Error(`Please specify a userID for this function.`)
-        } else {
-            schedules = await sdk.ok(sdk.all_scheduled_plans({user_id: userID, all_users:false}))
-        }
+        schedules = await sdk.ok(sdk.all_scheduled_plans({all_users:true}))
     }
     return schedules
 }
 
 const bulkDisableSchedules = async (allUsers: boolean, userID?: number): Promise<boolean> => {
-    const schedules = await getSchedules(allUsers=allUsers, userID=userID)
-    try {
-        schedules.forEach(async (s) => await sdk.ok(sdk.update_scheduled_plan(s.id, {enabled:false})))
-        // depending on whether allUsers is set to true or not we will change what we log to the console
-        console.log(`Successfully disabled all schedules ${allUsers? 'for all users' : `for user ${userID}`}`)
-        return true
-    } catch (e) {
-        throw new Error(`There was an error trying to disable schedules. Full error message: ${e}`)
+    if(userID || allUsers) {
+        const schedules = await getSchedules(allUsers=allUsers, userID=userID)
```

Comment:
This parameter passing doesn't look like valid JavaScript or TypeScript to me. Does this actually run?

Reply:
Just re-did the two schedule examples, am just passing userID and doing it with valid JavaScript/TypeScript so this should be changed and resolved with a working example when I push next 👍